python/pyarrow/_csv.pyx

```diff
@@ -1025,17 +1025,21 @@ cdef class WriteOptions(_Weakrefable):
     batch_size : int, optional (default 1024)
         How many rows to process together when converting and writing
         CSV data
+    delimiter : char, optional (default `'`)
```

Comment:
nit, but I think Python would use `"'"`, and char is not a type in Python (it should be str). And in any case, is the delimiter not `,`?

Reply:
See https://github.com/apache/arrow/blob/36a76914cd42606be7ef847d500af67c64ccef26/python/pyarrow/_csv.pyx#L278 for instance

Reply:
Yes, sorry I missed it. Corrected that now.